sympy/physics/mechanics/tests/test_removal_of_reaction_forces.py

```diff
@@ -0,0 +1,72 @@
+import sympy as sm
+import sympy.physics.mechanics as me
+
+def test_removal_of_reaction_forces():
+    """
+    This function test whether the reaction forces are removed from the
+    dynamic parts of (fr + frstar) and from the forcing vector.
+    It is a simple pendulum where the foot is moving in the x direction.
+    """
+    t = me.dynamicsymbols._t
+    N, A = me.ReferenceFrame('N'), me.ReferenceFrame('A')
+    O, P, P1 = me.Point('O'), me.Point('P'), me.Point('P1')
+    O.set_vel(N, 0)
+
+    m, l, g = sm.symbols('m l g')
+    q, x, y, u, ux, uy = me.dynamicsymbols('q x y u ux uy')
+    x1, ux1 = me.dynamicsymbols('x1 u1')
+
+    auxx, auxy, fx, fy, F = me.dynamicsymbols('auxx auxy fx fy F')
+
+    A.orient_axis(N, q, N.z)
+    A.set_ang_vel(N, u * N.z)
+    P1.set_pos(O, x1 * N.x)
+    P1.set_vel(N, ux*N.x + auxx*N.x + auxy*N.y)
+    P.set_pos(P1, -l * A.y)
+    P.v2pt_theory(P1, N, A)
+
+    BODY = [me.Particle('body', P, m)]
+    FL = [(P, -m * g * N.y), (P1, F*N.x + fx * N.x + fy * N.y)]
```

Comment:
Is the kwarg needed because there would be no way to know if F or fx is the reaction force associated with uax?

Reply:
> Is the kwarg needed because there would be no way to know if F or fx is the reaction force associated with uax?

This is my thinking at the moment, I see no way right now to get the reaction forces.